server/chat-plugins/room-faqs.ts

```diff
@@ -3,12 +3,50 @@ import {FS, Utils} from '../../lib';
 export const ROOMFAQ_FILE = 'config/chat-plugins/faqs.json';
 const MAX_ROOMFAQ_LENGTH = 8192;
 
-export const roomFaqs: {[k: string]: {[k: string]: string}} = JSON.parse(FS(ROOMFAQ_FILE).readIfExistsSync() || "{}");
+export const roomFaqs: {[k: string]: {[k: string]: RoomFaq}} = (() => {
```

Comment:
Why does this use IIFE?

Reply:
In theory I fine with this. Although if you're going to define a function for `convertFaq` you might just also define a function for `loadFaqs`